### PR TITLE
Allow `fontFace` to accept array of font face rules

### PR DESCRIPTION
**Pull request by taylorfsteele**
This PR adds support to the `fontFace` function to accept an array of font face rules that share the same font family name.

Example usage:
```
// text.css.ts
import { fontFace, style } from '@vanilla-extract/css';

const gentium = fontFace([
  {
    src: 'local("Gentium")',
    fontWeight: 'normal',
  },
  {
    src: 'local("Gentium Bold")',
    fontWeight: 'bold',
  },
]);

export const font = style({
  fontFamily: gentium,
});

// CSS
@font-face {
  src: local("Gentium");
  font-weight: normal;
  font-family: "text_gentium__fih47p0";
}
@font-face {
  src: local("Gentium Bold");
  font-weight: bold;
  font-family: "text_gentium__fih47p0";
}
.text_font__fih47p1 {
  font-family: "text_gentium__fih47p0";
}
```

### CODE REVIEW

**Review by supersabillon — APPROVED** [review]

File: packages/css/src/style.ts [hunk 1]
```diff
@@ -93,19 +93,37 @@ export function globalStyle(selector: string, rule: GlobalStyleRule) {
...
     quotes: 'double',
   })}"`;
 
+  const fontFamilyError = outdent`
+  This function creates and returns a hashed font-family name, so the "fontFamily" property should not be provided.
+
+  If you'd like to define a globally scoped custom font, you can use the "globalFontFace" function instead.
+`;
+
+  if (Array.isArray(rule)) {
```
Comment (graup) [inline packages/css/src/style.ts:110]: I think you can simplify this code by doing
```
const rules = Array.isArray(rule) ? rule : [rule];
```
and then just keeping one code path below.

**Comment by taylorfsteele** [comment]
@askoufis I saw you ran some CI checks but they were cancelled, anything I need to do on my end?

**Comment by askoufis** [comment]
> @askoufis I saw you ran some CI checks but they were cancelled, anything I need to do on my end?

Not sure why one of them was cancelled. The other one failed because the prettier formatting step failed.

**Comment by taylorfsteele** [comment]
> > @askoufis I saw you ran some CI checks but they were cancelled, anything I need to do on my end?
> 
> Not sure why one of them was cancelled. The other one failed because the prettier formatting step failed.

Gah, didn't even see the prettier one break, my bad. Let's see if this latest commit does the trick

**Review by mattcompiles — APPROVED** [review]

**Comment by mattcompiles** [comment]
@taylorfsteele Thanks for the PR bud 🍻

**Pull request merged**